GoogleForCreators/web-stories-wp
# Dashboard: Update SnackBar Text

## Context
The update in snackbar styles eliminates the 'title' area we'd been relying on to give context to errors being displayed, this updates messages in the dashboard to make sense with the new snackbar styles. 

## Summary
Cleans up error messages in dashboard now that I have feedback on text for errors and how to move forward. Catches snackbar implementation in dashboard up to changes made in design system. This will allow #4967 to _finally_ get merged (what a journey).

## Relevant Technical Choices

- custom zIndex given to snackbar in design system actually needs to be on the message not the container, that update is in here. 
- Now that we're just displaying an error message in the snackbar we don't need an object wrapping title and body for each error so simplifying that structure. 

## To-do

- Learned here while testing that there's no visual error handling for user settings - definitely edge case but should handle that when there's time. Right now it'll just fail silently, nothing happens. I will follow up with a bug ticket.

## User-facing changes

- Dashboard errors (and successful copy messages) should have new text matching this document:https://docs.google.com/spreadsheets/d/1PjaRp4bueQ3Cv14A7I696m0vjGCIX_kN9tIGh-V-Bkc/edit#gid=412530555 and follow styles of new snackbar component from design system. 

## Testing Instructions

- Block localhost after initially loading dashboard stories and try to do the following: 
  - Click on drafts or published stories on the top header, should see 'Unable to load stories'
  - Delete a story, see 'Unable to delete story' 
  - Rename a story, see 'Unable to rename story'
  - Duplicate a story, see 'Unable to duplicate story'
- Change line 110 in stories_controller.php to `$data['story_data'] = [];` to see this error: 
![Screen Shot 2021-02-10 at 10 04 16 AM](https://user-images.githubusercontent.com/10720454/107544348-5d990b00-6b87-11eb-9fd5-4f6da99218fa.png)
- Copy a story, see '[story name] has been copied to your clipboard'
- Go to settings, block localhost and try updating analytics id, uploading 1 logo, updating multiple logos. 


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #4967 
Fixes #6157
